src/test/java/edu/unc/lib/boxc/migration/cdm/services/PermissionsServiceTest.java

```diff
@@ -205,6 +205,147 @@ public void generateDefaultPermissionsWithForceFlagTest() throws Exception {
         }
     }
 
+    @Test
+    public void generateWorkPermissionsTest() throws Exception {
+        testHelper.indexExportData("grouped_gilmer");
+        Path permissionsMappingPath = project.getPermissionsPath();
+        var options = new PermissionMappingOptions();
+        options.setWithWorks(true);
+        options.setEveryone(UserRole.canViewMetadata);
+        options.setAuthenticated(UserRole.canViewMetadata);
+
+        service.generatePermissions(options);
+        assertTrue(Files.exists(permissionsMappingPath));
+
+        try (
```

Comment:
at this point we should probably make a helper method that reads the mapping file and returns a list of csvrecords, since we're repeating that so many times